_schemaV2.graphql

```diff
@@ -9922,6 +9923,29 @@ enum PartnersSortType {
 
 union PartnerTypes = ExternalPartner | Partner
 
+enum PhoneNumberErrors {
+  INVALID_COUNTRY_CODE
+  INVALID_NUMBER
+  TOO_LONG
+  TOO_SHORT
+}
+
+enum PhoneNumberFormats {
+  E164
+  INTERNATIONAL
+  NATIONAL
+  RFC3966
+}
+
+type PhoneNumberType {
+  countryCode: String
+  display(format: PhoneNumberFormats): String
+  error: PhoneNumberErrors
```

Comment:
**question (non-blocking):** Is this the pattern that other queries use to report errors in Metaphysics?

Reply:
honestly i don't know about patterns that metaphysics uses to report errors. I just added an enum with possible errors and change field name from `errorMessage` to `error`.

Reply:
Got it. I think we can launch anonymous submission with this implementation, and follow-up with another ticket if there is a pattern that we should follow.

@mzikherman is there a preferred pattern that Metaphysics queries should follow for reporting errors to clients?